test/libs/metrics/top_level_metrics_test.py

```diff
@@ -31,6 +39,42 @@
 ]
 
 
+def build_dataset(
```

Comment:
This looks like it largely duplicates what's in `_build_timeseries_dataframe`.  Maybe worth adding a comment why this is different and direction on how to consolidate the two in the future?

Reply:
I didn't see an obvious way to merge them because they each make a slightly bespoke DataFrame for MultiRegionDataset and the metrics output. I went the opposite direction and removed _build_timeseries_dataframe.